Code quality.md

```diff
@@ -0,0 +1,51 @@
+Being mindful of code quality is a desirable trait of every developer. At Infinum, code quality matters. With that in mind, we have come up with a few general rules that every Android developer should follow.
+
+### Lint
+
+Lint is a static code checker and is used to indicate suspicious and potentially harmful lines of code that do **not** follow certain style guidelines.
+
+Lint should always be enabled on release builds. In other words, every build that is given into the hands of our internal QA team or the client **must** be run through lint and other static code checkers on CI.
+
+For example, you might be working on an older project with outdated codebase. In that case, if static code checkers haven't already been set up, it's expected of you to set them up and create a [baseline](https://sites.google.com/a/android.com/tools/tech-docs/lint-in-studio-2-3#TOC-Creating-a-Baseline) if necessary. The entire new codebase that's written in the project **should be run through** static code checkers.
```

Comment:
Here the word *pass* is not used to tell we are running the check but to indicate the status of the check. The static analysis might *fail* or *pass*. I think we need to keep the wording here.

Reply:
Yes, keep _pass_ in that case.